app/services/debt/tuition/operation/no_discount.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require_relative '../tuition_handler'
+
+module Debt
+  module Tuition
+    module Operation
+      class NoDiscount < TuitionHandler
+        include Creation
+
+        def operate(params)
+          create_debt(params, description)
+        end
+
+        def fulfill?(_params)
+          true
+        end
+
+        private
+
+        def description
+          I18n.t('.tuition_management.tuition_debts.descriptions.no_discount')
```

Comment:
Bu description'ı translate etmek çok gereksiz. Çünkü kişi uygulamayı `en` dilinde kullanıyorsa harç borcu oluşturulurken açıklama ingilizce olacak fakat farklı bir kullanıcı bu açıklamayı türkçe olarak görüntülemek isteyecek. O bakımdan amaca hizmet etmiyor.

`description` alanına key eklersek örneğin `no_discount` ve bunu arayüzde translate edersek amaca ulaşmış oluruz. Açıklamaları standart hale getirmek açısından böyle bir değişiklik yapabiliriz ve description'a text eklemek yerine enum değeri olarak tutabiliriz.